torchani/extension/README.md

```diff
@@ -0,0 +1,44 @@
+# CUAEV
+CUDA Extension for AEV calculation.
+Performance improvement is expected to be ~3X for AEV computation and ~1.5X for overall training workflow.
+
+## Install
+In most cases, if `gcc` and `cuda` environment are well configured, runing the following command at `torchani/extension` directory will install the cuaev package.
+```bash
+pip install -e .
+```
+
+Notes for install on Hipergator
+```bash
+srun -p gpu --gpus=geforce:1 --time=01:00:00 --mem=10gb --pty -u bash -i   # compile may fail because of low on memery (when memery is less than 5gb)
+conda install pytorch torchvision cudatoolkit=10.0 -c pytorch              # make sure it's cudatoolkit=10.0
+module load cuda/10.0.130
+module load gcc/7.3.0
+pip install -e .
+```
+
+## Usage
+Pass `use_cuda_extension=True` when construct aev_computer, for example:
+```python
+cuaev_computer = torchani.AEVComputer(Rcr, Rca, EtaR, ShfR, EtaA, Zeta, ShfA, ShfZ, num_species, use_cuda_extension=True)
+```
+
+## Limitations
+Current implementation of CUAEV does not support pbc and force calculation.
+
+## Benchmark
+Benchmark of [torchani/tools/training-aev-benchmark.py](https://github.com/aiqm/torchani/tree/master/torchani/tools/training-aev-benchmark.py) on RTX 2080 Ti:
+
+|         ANI-1x          |     Without Shuffle     |         Shuffle         |
```

Comment:
Why are there different benchmarks for shuffle and without shuffle? I don't think shuffling batches should affect the AEVs at all, right?

Reply:
Yes, you are right. There is no big difference now for cuaev. But for python aev, there is still difference. I think it's not bad to keep it here, as it's already tested.